### PR TITLE
update static create methods for modules

### PR DESCRIPTION
**Pull request by Rahat-ch**

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -231,22 +231,20 @@ const bundler: IBundler = new Bundler({
...
-    let biconomyAccount = new BiconomySmartAccountV2(biconomySmartAccountConfig)
+    let biconomyAccount = await BiconomySmartAccountV2.create({
+      signer: wallet,
+      chainId: ChainId.POLYGON_MUMBAI,
+      bundler: bundler, 
+      entryPointAddress: DEFAULT_ENTRYPOINT_ADDRESS,
+      defaultValidationModule: module,
+      activeValidationModule: module
+    })
     biconomyAccount =  await biconomyAccount.init()
```
Comment (AmanRaj1608) [inline docs/quickstart.md:248]: can remove this line now

File: docs/tutorials/React_vite/sdk-integration.md [hunk 2]
```diff
@@ -174,15 +174,14 @@ async function setupSmartAccount() {
...
         signer: web3Provider.getSigner(),
         chainId: ChainId.POLYGON_MUMBAI,
         bundler: bundler, 
         entryPointAddress: DEFAULT_ENTRYPOINT_ADDRESS,
         defaultValidationModule: module,
         activeValidationModule: module
-      }
-      let biconomySmartAccount = new BiconomySmartAccountV2(biconomySmartAccountConfig)
+      })
       biconomySmartAccount =  await biconomySmartAccount.init()
```
Comment (AmanRaj1608) [inline docs/tutorials/React_vite/sdk-integration.md:185]: same here

File: docs/tutorials/React_vite/sdk-integration.md [hunk 3]
```diff
@@ -239,15 +237,11 @@ step-by-step explanation of what it does:
 -   `paymaster:` The paymaster used for handling payment processing. It is
     expected that the paymaster variable is defined elsewhere in the code.
 
-7. **`let biconomySmartAccount` = new
-   BiconomySmartAccount(biconomySmartAccountConfig):** Creates a new instance of
-   BiconomySmartAccount using the provided configuration.
-
-8. **`biconomySmartAccount` = await biconomySmartAccount.init():** Initializes
+7. **`biconomySmartAccount` = await biconomySmartAccount.init():** Initializes
```
Comment (AmanRaj1608) [inline docs/tutorials/React_vite/sdk-integration.md:240]: should be .create

File: docs/tutorials/nextjs/gaslesstransaction.md [hunk 4]
```diff
@@ -353,21 +353,20 @@ const connect = async () => {
...
         chainId: ChainId.POLYGON_MUMBAI,
         bundler: bundler, 
         paymaster: paymaster,
         entryPointAddress: DEFAULT_ENTRYPOINT_ADDRESS,
         defaultValidationModule: module,
         activeValidationModule: module
-      }
-      let biconomySmartAccount = new BiconomySmartAccountV2(biconomySmartAccountConfig)
+      })
       biconomySmartAccount =  await biconomySmartAccount.init()
```
Comment (AmanRaj1608) [inline docs/tutorials/nextjs/gaslesstransaction.md:370]: this should be removed too

**Review by AmanRaj1608 — COMMENTED** [review]
should replace all the .init, not needed anymore

**Review by livingrockrises — APPROVED** [review]

**Review by AmanRaj1608 — APPROVED** [review]
LGTM

**Review by tomarsachin2271 — APPROVED** [review]
LGTM

**Pull request merged**